cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -873,7 +873,7 @@ validatingRedeemersEx9 =
     [ (RdmrPtr Tag.Spend 0, (Data (Plutus.I 101), ExUnits 5000 5000)),
       (RdmrPtr Tag.Cert 1, (Data (Plutus.I 102), ExUnits 5000 5000)),
       (RdmrPtr Tag.Rewrd 0, (Data (Plutus.I 103), ExUnits 5000 5000)),
-      (RdmrPtr Tag.Mint 1, (Data (Plutus.I 104), ExUnits 5000 5000))
```

Comment:
looks like it is back to being 0 :confused:

Reply:
Not quite. It is just `missing 2-phase script witness` test case was added yesterday and I just rebased on master. The test case that was added relied on the md5 hashing. Just pushed, we'll see how CI copes with the fix